src/components/TodoList/TodoList.tsx

```diff
@@ -1,12 +1,40 @@
-/* eslint-disable */
 import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { useAppSelector } from '../../app/hooks';
+import { useDispatch } from 'react-redux';
+import { setCurrentTodo } from '../../features/currentTodo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  const dispatch = useDispatch();
+  const filter = useAppSelector(state => state.filter);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const filteredTodos = todos.filter(todo => {
+    switch (filter.status) {
+      case 'completed':
+        return todo.completed;
+      case 'active':
+        return !todo.completed;
+      default:
+        return true;
+    }
+  });
+  const visibleTodos = filteredTodos.filter(todo =>
+    todo.title.toLowerCase().includes(filter.query.trim().toLowerCase()),
+  );
 
-export const TodoList: React.FC = () => {
   return (
     <>
-      <p className="notification is-warning">
-        There are no todos matching current filter criteria
-      </p>
+      {false && (
+        <p className="notification is-warning">
+          There are no todos matching current filter criteria
+        </p>
+      )}
```

Comment:
The warning message for no matching todos is currently disabled with `false &&`. Consider enabling this feature to provide feedback to users when no todos match the current filter criteria.